x-pack/plugin/spatial/src/main/java/org/elasticsearch/xpack/spatial/proto/VectorTile.java

```diff
@@ -0,0 +1,5846 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+// Generated by the protocol buffer compiler.  DO NOT EDIT!
+// source: org/elasticsearch/xpack/spatial/vector_tile.proto
+
+package org.elasticsearch.xpack.spatial.proto;
+
+public final class VectorTile {
```

Comment:
Is there any chance we could generate this as part of the build and land it into a "generated sources" location?

Reply:
+1

Reply:
Right. What I did is to remove the prototype package all together. Instead I am using the generated files in the map box_vector_tile dependency.